packages/@aws-cdk/aws-eks/test/integ.eks-cluster.expected.json

```diff
@@ -3528,14 +3528,14 @@
               {
                 "Fn::GetAtt": [
                   "Cluster9EE0221C",
-                  "OpenIdConnectIssuer"
+                  "OpenIdConnectIssuerUrl"
```

Comment:
How did this happen?

Reply:
I think it happened because of this change:
https://github.com/aws/aws-cdk/pull/11109/files#diff-237d749c719902fe7de9dff86e143fdb0732cb90b5cfa6d6a7e6604a2d723923L74-R74